src/main/java/org/owasp/benchmark/score/parsers/SonarQubeJsonReader.java

```diff
@@ -197,9 +195,12 @@ public static int securityCategoryCWELookup(String secCat, String message) {
                         .equals(message)
                 || "Ensure that string concatenation is required and safe for this SQL query."
                         .equals(message)
+                || "Make sure using a dynamically formatted SQL query is safe here.".equals(message)
                 || "Make sure creating this cookie without the \"secure\" flag is safe here."
                         .equals(message)
                 || "Make sure that hashing data is safe here.".equals(message)
+                || "Make sure this weak hash algorithm is not used in a sensitive context here."
+                        .equals(message)
                 || "Make sure creating this cookie without the \"HttpOnly\" flag is safe."
                         .equals(message))) {
             System.out.println(
```

Comment:
I'm noticing that the old results from 5+ years ago: https://github.com/OWASP-Benchmark/BenchmarkJava/blob/master/results/Benchmark_1.2-sonar-Java-Plugin-v3.14-330.xml. Are far better than what I can generate today with the script you just provided and these updates to the SonarQubeJsonReader. Any clue why? Doesn't seem right to me.

Reply:
I also scored the Benchmark_1.2-sonarqube-v9.1.0.47736.json results I just generated and the scores aren't any different with or without these code changes to this Parser. Are the changes you made supposed to improve the score?  The ONLY score I'm seeing is: Weak Encryption Algorithm -100.00%	23.28%	76.72%, and all zeroes for every other category.  Is that what you are seeing too?